src/Symfony/Component/Messenger/DataCollector/MessengerDataCollector.php

```diff
@@ -79,6 +79,18 @@ public function reset()
         }
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    protected function getCasters()
+    {
+        $casters = parent::getCasters();
```

Comment:
or just `return [];` :)

Reply:
What about `ReflectionCaster::UNSET_CLOSURE_FILE_INFO` (added by parent) ?

Reply:
fair :)

Reply:
a small comment (or test) would probably help for future readings

Reply:
I've just added a small comment. Let me know if you had something else in mind.